lib/bozo/test_runners/nunit3.rb

```diff
@@ -57,6 +61,10 @@ def runner_args
         args << '--x86'
       end
 
+      unless !agents || agents.empty?
```

Comment:
this should be `unless !@agents || @agents.empty?` or you'll call the `agents` method with no arguments